src/uct/api/uct.h

```diff
@@ -204,6 +204,9 @@ typedef enum uct_atomic_op {
 #define UCT_IFACE_FLAG_ATOMIC_CSWAP32 UCS_BIT(22) /**< 32bit atomic compare-and-swap */
 #define UCT_IFACE_FLAG_ATOMIC_CSWAP64 UCS_BIT(23) /**< 64bit atomic compare-and-swap */
 
+#define UCT_IFACE_FLAG_ATOMIC32       UCS_BIT(24) /**< 32bit atomic operations */
```

Comment:
@hoopoepg i think we should change it only in UCP address packing functions, not in UCT API

Reply:
fixed